packages/loopring_v3/contracts/impl/libtransactions/AccountTransferTransaction.sol

```diff
@@ -142,6 +128,23 @@ library OwnerChangeTransaction
             // If the account has a wallet, use it to recover the account
             require(accountTransfer.walletHash != 0, "ACCOUNT_HAS_NO_WALLET");
 
+            // Verify authorization from the new owner
+            if (auxData.signature.length > 0) {
+                require(
```

Comment:
Across many files, we find this pattern of verifying either ECDSA signature or an approval hash. Maybe we can extract a common function?